eventual-cassandra/src/main/scala/com/evolutiongaming/kafka/journal/eventual/cassandra/SegmentNr.scala

```diff
@@ -49,20 +121,37 @@ object SegmentNr {
     }
   }
 
-  def apply(seqNr: SeqNr, segmentSize: SegmentSize): SegmentNr = {
+  @deprecated("use `journal` instead", "4.1.0")
+  def apply(seqNr: SeqNr, segmentSize: SegmentSize): SegmentNr = journal(seqNr, segmentSize)
+
+  /**
+    * Calculate segment number for `journal` table
+    */
+  def journal(seqNr: SeqNr, segmentSize: SegmentSize): SegmentNr = {
     val segmentNr = (seqNr.value - SeqNr.min.value) / segmentSize.value
     new SegmentNr(segmentNr) {}
   }
 
+  @deprecated("use `metaJournal` instead", "4.1.0")
   def apply(hashCode: Int, segments: Segments): SegmentNr = {
     val segmentNr = math.abs(hashCode.toLong % segments.value)
     new SegmentNr(segmentNr) {}
   }
 
+  /**
+    * Calculate segment number for `metajournal` table
+    */
+  def metaJournal(key: Key, segments: Segments): SegmentNr = {
+    val hashCode  = key.id.toLowerCase.hashCode
+    val segmentNr = math.abs(hashCode.toLong % segments.value)
+    new SegmentNr(segmentNr) {}
+  }
+
   def opt(value: Long): Option[SegmentNr] = of[Option](value)
 
   def unsafe[A](value: A)(implicit numeric: Numeric[A]): SegmentNr = of[Id](numeric.toLong(value))
 
+  @deprecated("use `fromSegmentSize` instead", "4.1.0")
```

Comment:
I didn't notice the `fromSegmentSize`, maybe?:
```suggestion
  @deprecated("use `allForSegmentSize` instead", "4.1.0")
```

Reply:
yeah, I renamed method after adding deprecation msg. Thanks